### PR TITLE
72019 - add content to product.txt file

### PR DESCRIPTION
**Pull request by abhijeet14992**
- Code changes to add content in Product.txt file
- Added unit test cases
- Added Function tests

### CODE REVIEW

File: UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.FulfilmentService/Services/FulfilmentDataService.cs [hunk 1]
```diff
@@ -85,11 +85,21 @@ public async Task<string> CreateExchangeSet(SalesCatalogueServiceResponseQueueMe
 
             if (aioConfiguration.IsAioEnabled)
             {
+                SalesCatalogueDataResponse catalogueDataResponse = (SalesCatalogueDataResponse)salesCatalogueEssDataResponse.Clone();
```
Comment (shailen14599) [inline UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.FulfilmentService/Services/FulfilmentDataService.cs:88]:
```suggestion
                SalesCatalogueDataResponse salesCatalogueEssDataResponseForAio = (SalesCatalogueDataResponse)salesCatalogueEssDataResponse.Clone();
```
Can we change name as suggested to make variable meaningful?

File: UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.FulfilmentService/Services/FulfilmentDataService.cs [hunk 2]
```diff
@@ -138,11 +148,25 @@ public async Task<string> CreateLargeExchangeSet(SalesCatalogueServiceResponseQu
 
             if (aioConfiguration.IsAioEnabled)
             {
+                var largeExchangeSetDataResponse = new LargeExchangeSetDataResponse()
```
Comment (shailen14599) [inline UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.FulfilmentService/Services/FulfilmentDataService.cs:151]:
```suggestion
                var largeExchangeSetDataResponseForAio = new LargeExchangeSetDataResponse()
```
Can we change name as suggested to make variable meaningful?

File: UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.Webjob.UnitTests/Services/FulfilmentDataServiceTest.cs [hunk 3]
```diff
@@ -611,6 +687,211 @@ public async Task WhenInvalidMessageQueueTrigger_ThenReturnsLargeMediaAndAioExch
             Assert.AreEqual("Large Media Exchange Set Is Not Created", result);
         }
 
+        [Test]
+        public async Task WhenValidMessageQueueTrigger_ThenReturnsLargeMediaAndStandardEncExchangeSetCreatedSuccessfully()
```
Comment (shailen14599) [inline UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.Webjob.UnitTests/Services/FulfilmentDataServiceTest.cs:691]: Looks like test case is written for AIO but in method name we can see StandardEncES. Can we change accordingly? Also can you add one liner summary for test case scenario as we are having multiple positive test cases?

**Review by shailen14599 — CHANGES_REQUESTED** [review]
Please verify unit tests for fake data.

File: UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.FulfilmentService/Services/FulfilmentDataService.cs (hunk 2 above)
Comment (abhijeet14992) [reply UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.FulfilmentService/Services/FulfilmentDataService.cs]: Resolved

File: UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.Webjob.UnitTests/Services/FulfilmentDataServiceTest.cs (hunk 3 above)
Comment (abhijeet14992) [reply UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.Webjob.UnitTests/Services/FulfilmentDataServiceTest.cs]: Modify method name and added description on test cases.

**Review by shailen14599 — APPROVED** [review]
LGTM

**Review by ray-sj — APPROVED** [review]
looks fine

**Pull request merged**